### ISSUE TITLE
Errno 13 during language pack update

### ISSUE THREAD

**Issue by radinamatic** [learningequality/ka-lite#4538]
Errno 13 during language pack update

**Branch**: 0.15, **ka-lite_0.15~b2-0ubuntu1_all.deb** installer  on Ubuntu 15.04

**Current Behavior**: 
On freshly installed 0.15 the first thing I did was to update English language pack: the process seemed to have finished correctly but with this error:

`Error during update: Error: [Errno 13] Permission denied: '/usr/share/kalite/data/khan/video_file_sizes.json'` 

![errno13](https://cloud.githubusercontent.com/assets/1457929/10176495/617a25a2-66f5-11e5-8333-69d2f5cf140a.png)

**Comment by radinamatic** [comment]
Same error appears after downloading Spanish language pack:

![errno13b](https://cloud.githubusercontent.com/assets/1457929/10177055/5cc35a4e-66f8-11e5-9145-85a51c7d0839.png)

##### However, the localized interface apparently works OK, once the language is changed:

![errno13c](https://cloud.githubusercontent.com/assets/1457929/10177104/bb2ee706-66f8-11e5-8f67-51a7f07c8e42.png)

**Comment by benjaoming** [comment]
This is not a problem with the .deb but a problem with the languagepacks getting unzipped to a system location rather than a user space location.

Did anything change?

**Comment by benjaoming** [comment]
The problem isn't this:

```
>>> from django.conf import settings
>>> settings.USER_WRITABLE_LOCALE_DIR
'/home/benjamin/.kalite/locale'
```

**Comment by benjaoming** [comment]
The problem is here:

```
>>> from kalite.updates import REMOTE_VIDEO_SIZE_FILEPATH
>>> REMOTE_VIDEO_SIZE_FILEPATH
'/usr/share/kalite/data/khan/video_file_sizes.json'
```

**Comment by benjaoming** [comment]
The problem is that when downloading locales, we initially wrote stuff here:

    settings.USER_WRITABLE_LOCALE_DIR

However, now that we also have `video_file_sizes.json` to worry about, we have to put that in a writeable location, too.

Not saying this issue is necessarily new.

**Comment by benjaoming** [comment]
I'm not sure I really understand why there's a singular `REMOTE_VIDEO_SIZE_FILEPATH` that gets overwritten by a languagepack?

Say we have English videos and a `video_file_sizes.json` for these videos. Then we fetch a Spanish / Castilian language pack and then the global `video_file_sizes.json` is overwritten by that.

@aronasorman ?

**Comment by aronasorman** [comment]
The assumption here is that the most recently downloaded language pack will have an updated `video_file_sizes.json`. Said files contains video file sizes for all languages.

But I agree that we can do better. I'll integrate video size into the content db as per [this issue](https://github.com/learningequality/ka-lite/issues/4505), so in the future we won't need this anymore.

**Comment by benjaoming** [comment]
@aronasorman can we remove `video_file_sizes.json` from the languagepacks then?

As a quick-fix I can just disable the part that moves the file from the zip to the system location, since this is the broken part....

Would this be okay? @rtibbles ?

**Comment by aronasorman** [comment]
No problem with that. If we don't have the latest file sizes we just default to 14.3 MB. And yes, this file will be removed in the future.

**Comment by jinchuika** [comment]
I can confirm this on 0.15b2, trying to install Spanish language pack.

**Comment by benjaoming** [comment]
@jinchuika new release coming up in new PPA:

https://launchpad.net/~learningequality/+archive/ubuntu/ka-lite-updates

**Comment by rtibbles** [comment]
Fixed in #4546
